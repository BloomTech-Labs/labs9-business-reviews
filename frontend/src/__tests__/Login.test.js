import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import LoginComponent from '../components/Login';

describe('<LoginComponent />', () => {
  it('renders', () => {
    shallow(<LoginComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<LoginComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
  it('should contain NavBar', () => {
    let app = shallow(<LoginComponent />);
    expect(app.find("NavBar").length).toBe(1);
  });
  it('should contain GatedSignIn', () => {
    let app = shallow(<LoginComponent />);
    expect(app.find("GatedSignIn").length).toBe(1);
  });
});
