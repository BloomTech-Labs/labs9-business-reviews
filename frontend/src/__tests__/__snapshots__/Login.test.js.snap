// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginComponent /> matches snapshot 1`] = `
<div>
  <NavBar />
  <styled.div>
    <GatedSignIn />
  </styled.div>
</div>
`;
